scriptmodules/supplementary/emulationstation.sh

```diff
@@ -17,7 +17,8 @@ function depends_emulationstation() {
     getDepends \
         libboost-locale-dev libboost-system-dev libboost-filesystem-dev libboost-date-time-dev \
         libfreeimage-dev libfreetype6-dev libeigen3-dev libcurl4-openssl-dev \
-        libasound2-dev cmake libsdl2-dev libsm-dev
+        libasound2-dev cmake libsdl2-dev
```

Comment:
(and libsm-dev is installed with sdl1.2 etc, so it exists on the system I use to build binaries. A user installing just emulationstation on raspbian jessie lite noticed this). Actually, the cmakefiles for ES could be fixed for rpi so X is ignored - it's pulled in by one of the configs)